Source/Example Measurements/NI-SCOPE Acquire Waveform/README.md

```diff
@@ -0,0 +1,28 @@
+## NI-SCOPE Acquire Waveform
+
+This is a MeasurementLink example that acquires waveforms from an NI SCOPE from the specified pins.
```

Comment:
I would remove the part about pins.
`This is a MeasurementLink example that acquires waveforms from an NI SCOPE.`